src/Host/Client/Impl/Extensions/AboutHostExtensions.cs

```diff
@@ -2,18 +2,19 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Reflection;
 using Microsoft.Common.Core;
 using Microsoft.R.Host.Protocol;
 
 namespace Microsoft.R.Host.Client {
     public static class AboutHostExtensions {
-        private static readonly Version _localVersion;
+        private static Version _localVersion;
```

Comment:
Updated and remove the service i had added to handle this.

---
In reply to: [111816503](https://github.com/Microsoft/RTVS/pull/3419#discussion_r111816503) [](ancestors = 111816503)